routes/lang/en-application.json

```diff
@@ -410,5 +410,11 @@
   "app.searchByMail": "Search by email",
   "app.maxfileuploadsize": "Maximum upload file size: %{size}.",
   "app.uploadsizeerror": "File size limit exceeded. Expected no more than %{max}, received %{size}.",
+  "app.fileuploaderror": "File upload failed.",
+  "app.cannotfindbower": "Cannot find expected bower.json file in the plugin root, please check the structure of your zip file and try again.",
+  "app.unrecognisedplugin": "Unrecognized plugin - a plugin should have a bower.json file.",
+  "app.unrecognisedpluginforpackage": "Unrecognized plugin type for package %{package}.",
+  "app.incompatibleframework": "This plugin is incompatible with version %{framework} of the Adapt framework.",
+  "app.versionexists": "This version of this plugin already exists, please update the version number and try again.",
```

Comment:
Is it better to say something like "You already have this version of the plugin installed" so it's more useful to non-dev users as well?

Reply:
That is definitely a better way of saying that :+1: